src/crypto/OutgoingRoomKeyRequestManager.js

```diff
@@ -198,15 +198,15 @@ export default class OutgoingRoomKeyRequestManager {
                         // (We also don't want to wait for the response from the server
                         // here, as it will slow down processing of received keys if we
                         // do.)
-                        this._sendOutgoingRoomKeyRequestCancellation(
+                        return this._sendOutgoingRoomKeyRequestCancellation(
```

Comment:
Please don't change code without changing the corresponding comments to make them not be lies. [here](https://github.com/matrix-org/matrix-js-sdk/pull/621/files#diff-d8c620291b1726321a300d1e99bb1256R198) and [here](https://github.com/matrix-org/matrix-js-sdk/pull/621/files#diff-d8c620291b1726321a300d1e99bb1256R137)